### ISSUE TITLE
Tests with errors don't fail when you pick Run Tests, only Debug Tests

### ISSUE THREAD

**Issue by nightblade9** [MikeSchulze/gdUnit3#189]
**The used Godot version:** 3.4.stable.official

**OS including version:** Windows 10

**Describe the bug**
Create a test with code that causes an error inside; run it. gdUnit3 reports the test as successful. (If you debug the test, Godot stops at the error and tells you what the problem is.)

This makes it easy to write tests that appear to work, but don't.

**Steps to Reproduce**
Add this test-file and run it in Godot; this is an example of how to simulate a failure. (Another type of failure is to call a method with the wrong number of arguments.)

```gdscript
extends GdUnitTestSuite

func test_fails():
  var x = 0
  var y = 1/x
  assert_int(x).is_equal(0)
```

For a similar failure, you can call `"test".subst(2, 3, 4)` instead of `1/x`.

**Comment by MikeSchulze** [comment]
Thanks for reporting this, I will look into this as soon as possible.

**Comment by MikeSchulze** [comment]
Closed as duplicate to #112

**Comment by nightblade9** [comment]
Sorry, took a look for related issues but didn't see #112.
